Protocol.md

```diff
@@ -177,8 +177,7 @@ ReactiveSocket implementations may provide their own validation at the metadata
 
 Specific Frame Types MAY contain Metadata. If that Frame Type supports both Data and Metadata, the optional Metadata header MUST be included. This metadata header is between the Frame Header and any payload.
 
-Metadata Length MUST be less than or equal to the Frame Length minus the length of the Frame Header.
-If Metadata Length is greater than this value, the entire frame MUST be ignored.
+Metadata Length MUST be equal to the Frame Length minus the sum of the length of the Frame Header and the length of the Frame Payload, if present. If Metadata Length is not equal to this value, the entire frame MUST be ignored.
```

Comment:
I am fine with this clarification. I understand the spec the same way before and after.